hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamer.java

```diff
@@ -2291,7 +2291,7 @@ public void testCsvDFSSourceNoHeaderWithoutSchemaProviderAndWithTransformer() th
       testCsvDFSSource(false, '\t', false, Collections.singletonList(TripsWithDistanceTransformer.class.getName()));
     }, "Should error out when doing the transformation.");
     LOG.debug("Expected error during transformation", e);
-    assertTrue(e.getMessage().contains("cannot resolve 'begin_lat' given input columns:"));
+    assertTrue(e.getMessage().contains("Column 'begin_lat' does not exist. Did you mean one of the following?"));
```

Comment:
Azure CI runs this test in Spark 3.2.  This error message after change only works for Spark 3.3.

Reply:
Thank you! I've reverted changes in a separate commit d6384c6e0cadc515ff59fe46a15277102db1b1bd

Reply:
Now check is looking for any of the two mentioned messages.

Reply:
@yihua Azure CI is successful now. Could you, please, mark your request change as resolved?